packages/teleterm/src/sharedProcess/ptyHost/server/ptyEventsStreamHandler.ts

```diff
@@ -9,9 +9,11 @@ import {
   PtyServerEvent,
 } from '../../api/protogen/ptyHostService_pb';
 import { PtyProcess } from '../server/ptyProcess';
+import Logger from 'teleterm/logger';
 
 export class PtyEventsStreamHandler {
   private readonly ptyId: string;
+  private logger = new Logger('PtyEventsStreamHandler');
```

Comment:
Since we know the ptyId upfront, could we instead do something like 

```typescript
new Logger(`PtyEventsStreamHandler (id: ${this.ptyId})`)
```

in the constructor? This way we won't have to repeat "stream for pty" in each logger invocation.

Reply:
Good idea, done